english/explore-outside/journey-to-flaming-tower/walk/sailboat/find-owner/examine-wallet/wallet-contents.md

```diff
@@ -0,0 +1,3 @@
+In the wallet you find several pictures, one of which is of a happy-looking family on the sailboat. Also, you find a picture ID with the person's name and address.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.